### PR TITLE
Add flakes to the final command to directly enter into the developer …

### PR DESCRIPTION
**Pull request by star-gazer111**
…environment while building locally

### CODE REVIEW

**Comment by star-gazer111** [comment]
[Fix](https://github.com/fedimint/fedimint/issues/4777)

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ git clone https://github.com/your-username/fedimint.git
 Then enter the nix developer environment.
 
 ```bash
-nix develop
+nix --extra-experimental-features nix-command --extra-experimental-features flakes develop
```
Comment (dpc) [inline README.md:80]: We should change the installer recommendation to one from https://github.com/DeterminateSystems/nix-installer , so the users don't need to worry about enabling flakes. It is generally considered much better than the official one, and

Comment (star-gazer111) [reply README.md:80]: Hmm, that's even better I suppose. So should I make the appropriate changes in the Nix Installation section?

Comment (dpc) [reply README.md:80]: Go for it. Thanks!

**Pull request closed without merging**